### PR TITLE
Initial checkin of MassTransactorAction

### PR DESCRIPTION
**Pull request by mbcomer**
This is the initial checkin of the code for MassTransactorAction:

https://unofficialsf.com/create-delete-or-update-tens-of-thousands-of-records-at-a-time-in-flow-with-the-new-mass-transactor-action/

### CODE REVIEW

**Comment by alexed1** [comment]
this is great. I'd love to see a video that essentially shows the side-by-side difference of trying to do something synchronously vs doing it with this.

**Comment by adwhite75** [comment]
Thanks Alex - I still want to do a highlight of this soon!

**Comment by alexed1** [comment]
Hi just following up on this. Really looking forward emerging miss him I left notes on a few small changes I'd like for consistency and clarity￼

File: flow_action_components/MassTransactorAction/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# MassTransactorAction
+
+This Invocable allows Flow (or Proces Builder) to perform large DML operations by leveraging Batch Apex.
+
+See [this article](https://unofficialsf.com/create-delete-or-update-tens-of-thousands-of-records-at-a-time-in-flow-with-the-new-mass-transactor-action/) for more information.
+
+This is the first release of this component and it does have a few limitations. We do plan to do more releases to address some of these.
+- This is currently not well bulkified for use in autolaunched Flow or PB where batches of records may be present. There are some complexities in making this work, but we do plan to release an improved version with basic bulkification shortly.
+- Due to Salesforce limitations around the number of running and queued batches in an org, this might not scale to all use-cases. Proper testing is recommended!
+- Right now we have only implemented one OnFinish action, a simple "email on complete" action. We may implement more, however you can use the MassTransactorBatchHelper.OnFinishAction interface to roll your own!
```
[the remark this answers is not in the document]
Comment (adwhite75) [reply flow_action_components/MassTransactorAction/README.md:10]: Hey Alex I can update the Future Ideas section to mention we have a ideas for more 'onfinish' actions but I'm not quite sure what else needs to be said other than the email

File: flow_action_components/MassTransactorAction/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# MassTransactorAction
```
[the remark this answers is not in the document]
Comment (alexed1) [reply flow_action_components/MassTransactorAction/README.md:1]: @jdpond just checking on this name change suggestion

File: flow_action_components/MassTransactorAction/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (alexed1) [reply flow_action_components/MassTransactorAction/README.md:10]: not talking about future stuff.... I was just unclear on how the Email on Complete part works and what 'OnFinish' means and how the interface works.

**Comment by mbcomer** [comment]
Not a problem, I can take care of this ASAP!

**Comment by alexed1** [comment]
Hi @mbcomer, any update?

**Comment by mbcomer** [comment]
@alexed1 Sorry for the delay, just too much life and not enough time lol! Rename all taken care of.

**Review by alexed1 — APPROVED** [review]
Great!

**Pull request merged**